### DISCUSSION TITLE
Webhook listener setup on production

### DISCUSSION THREAD

**Discussion by maulikbafalipara** [pay-rails/pay#665] [General] [answered]
Hello all,
I am done with all the basic configuration with pay gem and stripe details and it's working fine on development env.

now I want to move on to production with these new features,
I did all the basic configurations that are needed, but I don't know how to subscribe for webhook in development, I am using ```stripe listen --forward-to http://localhost:3000/pay/webhooks/stripe``` this for development but on production how can I start,
do I need to use the same command in demon mode?

can anyone please guide me?

Thank in advance

**Comment by excid3** [comment, marked as answer]
You simply create a webhook in the Stripe dashboard that sends to your production domain.

The Stripe CLI is only for development.

**Reply by rrishabh** [reply]
Is there any other setup required to run the webhooks in the server so that it runs continuously?

**Reply by excid3** [reply]
Nope. Production goes directly to your Rails app.
